README.md

```diff
@@ -60,91 +61,94 @@ Website is accessible at https://athityakumar.github.io/tvseries/index.html
 
 <br>Feel free to add more series' to the list by sending Pull Requests.
```

Comment:
Remove all the `<br>` tags, please.

Reply:
Sorry for the carelessness .
Please tell me if its fine now.